src/components/pages/PendingApplications/PendingApplication.js

```diff
@@ -103,70 +91,74 @@ const PendingApplications = () => {
     setModalIsVisible(true);
   };
 
-  useEffect(() => {
-    const getPendingApps = () => {
-      axiosWithAuth()
-        .post('/application')
-        .then(res => {
-          res.data.users.forEach(row => {
-            row.hasOwnProperty('accepting_new_mentees')
-              ? (row.role_name = 'mentor')
-              : (row.role_name = 'mentee');
-          });
-          setApplications(
-            res.data.users.map(row => ({
-              key: row.profile_id,
-              name: row.first_name + ' ' + row.last_name,
-              role: (
-                <Tag color={row.role_name === 'mentor' ? 'blue' : 'purple'}>
-                  {row.role_name}
-                </Tag>
-              ),
-              // hard-coded "Date Submitted" field values because DS field for date does not exist.
-              date: Date(row.updated_at).slice(0, 15),
-              status: (
-                <Tag
-                  color={
-                    row.validate_status === 'approved'
-                      ? 'green'
-                      : row.validate_status === 'pending'
-                      ? 'orange'
-                      : 'red'
-                  }
-                >
-                  {row.validate_status}
-                </Tag>
-              ),
-              button: (
-                <Button
-                  style={{
-                    backgroundImage:
-                      'linear-gradient(-180deg, #37AEE2 0%, #1E96C8 100%)',
-                    borderRadius: '.5rem',
-                    boxSizing: 'border-box',
-                    color: '#FFFFFF',
-                    display: 'flex',
-                    fontSize: '16px',
-                    justifyContent: 'center',
-                    cursor: 'pointer',
-                    touchAction: 'manipulation',
-                  }}
-                  type="primary"
-                  id={row.profile_id}
-                  onClick={() => showModal(row.profile_id)}
-                >
-                  Review Application
-                </Button>
-              ),
-            }))
-          );
-        })
-        .catch(err => {
-          console.log(err);
+  const getPendingApps = () => {
+    axiosWithAuth()
+      .post('/application')
+      .then(res => {
+        res.data.users.forEach(row => {
+          row.hasOwnProperty('accepting_new_mentees')
+            ? (row.role_name = 'mentor')
+            : (row.role_name = 'mentee');
         });
-    };
+        setApplications(
+          res.data.users.map(row => ({
+            key: row.profile_id,
+            name: row.first_name + ' ' + row.last_name,
+            role: (
+              <Tag color={row.role_name === 'mentor' ? 'blue' : 'purple'}>
+                {row.role_name}
+              </Tag>
+            ),
+            date: (row.updated_at ? row.updated_at : row.created_at).slice(
```

Comment:
Will the difference between the updated date and created date be clear? I thought I remember in one of the last meetings that the date should be one or the other, depending on the status of the user?

Reply:
Date cleanup has a follow-up ticket.